internal/cli/cmd/dac/setup/golang.go

```diff
@@ -0,0 +1,33 @@
+// Copyright 2024 The Perses Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package setup
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+)
+
+func (o *option) setupGo() error {
+	if _, err := os.Stat("go.mod"); os.IsNotExist(err) {
```

Comment:
could you add the same check on cue side for the `cue.mod` folder ?

Reply:
done